images/access_setup/content/bin/test.sh

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env bash
```

Comment:
test.sh is not documented in README. I would like to see what the purpose is and when it is to be run.
Obviously running it after the couple of scripts here would not be enough.
Also it seems similar in purpose to [this script](https://github.com/openshift-pipelines/pipelines-service/blob/main/ckcp/test.sh) and think that it may make sense to unify and possibly move the script to a common place (something we may also want in our CI) that may be called by a script here providing the right parameters

Reply:
Removed